bootstrap.bat

```diff
@@ -1,4 +1,7 @@
-nuget restore
+@echo off
+
+msbuild /t:restore

```

Comment:
I noticed a lot of nuget references in your output. Is it possible that the version of nuget you're using is incompatible? When looking for a nuget for 2017, I came across an article that stated the command line nuget was deprecated and being phased out by being integrated into msbuild